examples/hooks/components/HitsPerPage.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useCallback } from 'react';
+import { useHitsPerPage, UseHitsPerPageProps } from 'react-instantsearch-hooks';
+import { cx } from '../cx';
+
+export type HitsPerPageProps = React.ComponentProps<'div'> &
+  UseHitsPerPageProps;
+
+export function HitsPerPage(props: HitsPerPageProps) {
+  const { items, refine } = useHitsPerPage(props);
+  const { value: currentValue } =
+    items.find(({ isRefined }) => isRefined) || {};
+
+  const handleChange = useCallback(
```

Comment:
is this required? the refine callback should be static already no?

Reply:
Indeed, we don't need this overhead. Watching the reference is more costly than creating a new function reference here.

Reply:
removed in https://github.com/algolia/react-instantsearch/pull/3205/commits/ebb0c124098660e61bd1293d1d145f93ed7a94dc